buildtest/cli/__init__.py

```diff
@@ -832,6 +832,13 @@ def config_menu(subparsers):
         action="store_true",
         help="List compiler details in YAML format",
     )
+    compilers.add_argument(
+        "-m",
+        "--modulepath",
+        type=str,
+        nargs="+",
+        help="List modulepath to search for compiler directories",
```

Comment:
Say `Specify a list of directories to search for modules via MODULEPATH when detecting compilers`